docs/testplan/Chassis-fabric-test-plan.md

```diff
@@ -21,11 +21,74 @@ The scope of this test plan is as follows:
 
 The current SW design for fabric does not cover events like card insertion/removal or reboots. This test plan depends on fabric counter cli support (work in progress).
 
-# Test Setup
+## Test Setup
 
 These test cases will be run in the proposed [T2 topology](https://github.com/Azure/sonic-mgmt/blob/master/ansible/vars/topo_t2.yml). It is assumed that such a configuration is deployed on the chassis.
 
-tbinfo will be populated with the number of fabric links per forwarding ASIC that are expected to be up.
+These test cases will compare the following two sets of data on a chassis:
+* Expected fabric link status
+* Current fabric link status
+
+The following section describes how to store and process the expected fabric link status for testing.
+
+
+![](Img/Sonic_Fabric_Link_Testing_Proposal.png)
+
+The above diagram illustrates an example system under test. Every forwarding ASIC is connected to every fabric ASIC.
+
+The expected fabric link status is stored in the last section of testbed.yaml file, which is called fabric_link_toplogy. The expected status is provided per host and per ASIC, and the format is as follows:
+
+```
+host:
+  asic:
+      link_id:
+         status: <connected/not_connected>
+         peer_asic: <asic id>
+         peer_id: <link id>
+         peer_status: <connected/not_connected>
+```
+
+Initially, testbed.yaml will contain fabric link status, such as whether or not links are connected. In the future, this information may be extended to contain fabric link connection information.
```

Comment:
Please check if this information can be added in the seperate yaml file for each testbed.

Reply:
The the fabric link map information is generated dynamically when run the test based on which slot the card is inserted in the system ,  which type of cards ( sku ), and all these information is in the testbed.yaml file now, and we can get the information while generating the testbed.yaml file and the same time update the fabric topology into it. If they are in separate files, then we will need to get the information from processing  the testbed.yaml and then generate the fabric topology into a different files, this can be done , but it will taking longer time also need to parse the same information several times . So we want to double check here if we still want to have the fabric topology information in a separate yaml file ? thank you !

Reply:
Discussed with Arvind, the information will be added in separate yaml files for each type of cards.